src/scripts/glue_jobs/access_requirement_snapshot.py

```diff
@@ -0,0 +1,109 @@
+"""
+The job take the access requirements snapshot data from S3 and process it.
+Processed data stored in S3 in a parquet file partitioned by the date (%Y-%m-%d pattern) of the snapshot timestamp.
+"""
+
+import sys
```

Comment:
we might need an integration test that adds one of each type to verify each type.